### PR TITLE
Blog app - Add forms

### PR DESCRIPTION
**Pull request by kidde60**
### Blog app - Add forms.
**On this project I:**
- Create a method called current_user in your ApplicationController to make current user data available to all controllers.
It will return the first user from the database.
- Create forms to perform the following functions:
- Creates a Post on behalf of the current_user (use the method that you created in your ApplicationController).
- Create a comment on behalf of the current_user (use the method that you created in your ApplicationController).
- Allow Users to add likes to Posts.

### CODE REVIEW

File: app/controllers/likes_controller.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+    @like = Like.new
+  end
+
+  def create
+    @like = Like.new(post_id: params[:id], author: current_user)
+    return unless @like.save
+
+    redirect_to user_post_path(current_user.id, params[:id])
+  end
+end
```
Comment (julie-ify) [inline app/controllers/likes_controller.rb:12]: - [ ] I am unable to like a post. It throws an error. You can rectify this by going to Like model and change `author` to `post`

  ```
  post.increment!(...)
  ```

  image

**Review by julie-ify — CHANGES_REQUESTED** [review]
## Hi @kidde60,

Good job so far!

## Highlights!
- Use of first user for creating resources ✔️ 

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@julie-ify` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/controllers/likes_controller.rb (hunk 1 above)
Comment (kidde60) [reply app/controllers/likes_controller.rb:12]: Thanks a lot for the correction

**Review by ichala — APPROVED** [review]
Hi Student  ,
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:  
 
68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f6c3366516333614356304871707a5763552f67697068792e676966

# To Highlight 💪🏻

- Changes Implemented ✔️ 


 Good job implementing the requirements  :+1: Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Optional Suggestions  :

- N/A


**********
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag  ichala in your question so I can receive the notification.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**